app/controllers/beers_controller.rb

```diff
@@ -0,0 +1,55 @@
+class BeersController < ApplicationController
+  def new
+    @beer = Beer.new
+  end
+
+  def create
+    @beer = Beer.new(beer_params)
+    if @beer.save
+      flash[:notice] = "Beer was saved"
+      redirect_to 'beer#show'
+    else
+      flash[:notice] = "Beer not added"
+      render 'new'
+    end
+  end
+
+  def index
+    @beers = Beer.page(params[:page]).order(:name)
+  end
+
+  def show
+    @beer = Beer.find(params[:id])
+  end
+
+  def edit
+    @beer = Beer.find(params[:id])
+  end
+
+  def update
+    @beer = Beer.find(params[:id])
+    if @beer.update(beer_params)
+      flash[:notice] = "Beer updated"
+      redirect_to beers_path(@beer)
+    else
+      flash[:notice] = "Beer was not updated"
+      render :edit
+    end
+  end
+
+  def destroy
+    @beer = Beer.find(params[:id])
+    if @beer.destroy
+      flash[:notice] = "Beer deleted."
+    else
+      flash[:notice] = "Beer was not deleted"
```

Comment:
Minor point. Line 45 does not end in a period and line 43 does. Make sure they are all consistent with other controllers.